### ISSUE TITLE
icon in the menu button is not hidden

### ISSUE THREAD

**Issue by derekjackson-das** [danny-avila/LibreChat#3821]
**Issue Description**:
The menu button icon for the chat menu is not hidden from assistive technology, leading screen readers to announce the icon as content. Since the button is already named by its aria-label attribute, the icon should be hidden to avoid redundancy and confusion for users relying on screen readers.

Related to #3823 

Same issue in #3819 

**WCAG Criteria**:[1.1.1 Non-text Content (Level A)](https://www.w3.org/WAI/WCAG21/Understanding/non-text-content)

**Screenshot or Screen Recording:**
<img width="1669" alt="Screenshot 2024-08-28 at 11 55 26 AM" src="https://github.com/user-attachments/assets/a983fe76-2bbc-40df-b455-0dc24b863c8e">

**Notes**: Add the aria-hidden attribute to the svg tag with a value of "true". The icons in these buttons should be hidden unless they provide essential information not conveyed by the button's label.
